examples/test/tsconfig.json

```diff
@@ -4,6 +4,7 @@
   "compilerOptions": {
     "noEmit": true,
     "skipLibCheck": true,
-    "resolveJsonModule": true
+    "resolveJsonModule": true,
+    "esModuleInterop": true
```

Comment:
'esModuleInterop' makes typescript compiler treat commonjs similar to ES6 modules. 

I made the changes in all tsconfig.json. This way we abandon usage like `import * as Faye from 'faye';` which esbuild does not support for commonjs module. Even if there is no change of code here in examples/test, it is still good to use it.